src/test/java/org/vertx/eventbus3/bridge/TestServer.java

```diff
@@ -0,0 +1,73 @@
+package org.vertx.eventbus3.bridge;
+
+import org.vertx.java.core.Handler;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TestServer extends Thread {
+
+  Handler<String> lineHandler;
+
+  private void process(String line) {
+    if (line.startsWith("INFO: ")) {
+      if (lineHandler != null) {
+        lineHandler.handle(line.substring(6));
+      } else {
+        System.err.println("Unexpected: " + line.substring(6));
+      }
+    }
+  }
+
+  @Override
+  public void run() {
+    try {
+
```

Comment:
My idea was that if you download the test jar from maven and want to reproduce the running of tests then the dependency would be there but i'm totally fine to move it out.